src/Component/Locations.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios'
+class Locations extends React.Component {
+    constructor(){
+        super();
+        this.state = {
+            cities:[],
+            showCity: false
+        }
+    }
+
+    ghibliCities = async () =>{
```

Comment:
I suggest naming functions based upon what they do. Wouldn't getCities or getLocations be more appropriate? It helps other developers read what's happening in your code